### ISSUE TITLE
transform scale is incorrectly parsed when only 'x' is present

### ISSUE THREAD

**Issue by pekim** [lafriks/go-svg#1]
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/transform#scale
> The `scale(<x> [<y>])` transform function specifies a scale operation by `x` and `y`. If `y` is not provided, it is assumed to be equal to `x`.

https://github.com/lafriks/go-svg/blob/58ecbfca0208034d921c78d71cc0ac474c0e8f2f/parse.go#L83
I believe that
```go
		if ln == 1 {
			m1 = m1.Scale(c.points[0], 0)
```
should be
```go
		if ln == 1 {
			m1 = m1.Scale(c.points[0], c.points[0])
```

**Comment by pekim** [comment]
It looks like `translate` has the same issue.
https://github.com/lafriks/go-svg/blob/58ecbfca0208034d921c78d71cc0ac474c0e8f2f/parse.go#L63

**Comment by lafriks** [comment]
Fixed in v0.4.1 release
